README.md

```diff
@@ -2,7 +2,28 @@
 
 ## Opis projektu
 
-Przykładowa strona sklepu zakodowana za pomocą darmowego szablonu w celu nauki kodowania stron i nauki pracy w zespole.
+### Strona sklepu połączona z blogiem. Oparta na Bootsrap a więc łatwa w rozbudowie, elastyczna. Lekka, bez obciążających pluginów. Zgoda z RWD.
+
```

Comment:
To nagłówek, a jest długi jak paragraf ;)

Reply:
Poprawione